sources/hooks/conf_regen/98-postfix_emailpoubelle

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 source /usr/share/yunohost/helpers
+app=$YNH_APP_INSTANCE_NAME
```

Comment:
Uuuh i'm confused how this is supposed to fix the issue x_X 

This is a regenconf hook ran by the core, I don't think it feeds any YNH_APP_INSTANCE_NAME variable because regen conf hooks are not related to any specific app ?